app.py

```diff
@@ -18,14 +18,20 @@ def Onsubmit():
     url = f'http://api.openweathermap.org/data/2.5/weather?q={place}&APPID={API_KEY}&units=metric'
     weather_info = requests.get(url).json()
     
+    
     weather = {
         'place' : place,
-        'temparature' : weather_info["main"]["temp"],
+        'description' : weather_info["weather"][0]["description"],
+        'icon' : weather_info["weather"][0]["icon"],
+        'temperature' : weather_info["main"]["temp"],
+        'pressure' : weather_info["main"]["pressure"],
+        'humidity' : weather_info["main"]["humidity"],
+        'wind_speed' : weather_info["wind"]["speed"],
+        'clouds' : weather_info["clouds"]["all"],
+        'country' : weather_info["sys"]["country"]
     }
     
-    
-    return render_template('bulmacard.html', weather=weather)
-
+    return render_template('bulmacard.html', weather = weather)
```

Comment:
If you look at the above two lines, there is no logic change. It is just a spaces difference. So the PR should not contain these type of changes

Reply:
Yeah, there is no logic those lines. i definitely change that.

Reply:
This is happening often. Can you find a solution for this?

Reply:
Yeah.. that was correct subbu.